package.json

```diff
@@ -24,35 +24,35 @@
   ],
   "dependencies": {
     "hyphenate-style-name": "^1.0.0",
-    "lodash.omit": "^3.0.0",
-    "matchmedia": "^0.1.1",
-    "object-assign": "^2.0.0"
+    "matchmedia": "^0.1.2",
+    "object-assign": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^0.12.0 || ^0.13.0 || ^0.14.0"
+    "react": "^0.14.0"
```

Comment:
@pekeler The sample code is for the latest version, but that doesn't mean you have to drop support for old versions and break people's existing installs